src/types.ts

```diff
@@ -60,3 +60,16 @@ export interface ExtendedLog extends providers.Log {
   from?: Address;
   to?: Address;
 }
+
+export interface SimulateBody {
```

Comment:
Please leave a comment with a Link to the Tenderly docs so future devs know this isn't our custom type and can't be changed.
https://docs.tenderly.co/simulations-and-forks/simulation-api/using-simulation-api